modules/skybrowser/skybrowsermodule.cpp

```diff
@@ -0,0 +1,692 @@
+                "string, double, double",
+                "Takes the identifier of a sky browser or a sky target and equatorial "
+                "coordinates Right Ascension and Declination. The target will animate to "
+                "this coordinate and the browser will display the coordinate."
+            },
+            {
+                "setVerticalFov",
+                &skybrowser::luascriptfunctions::setVerticalFov,
+                "string, float",
+                "Takes an identifier to a sky browser or a sky target and a vertical "
+                "field of view. Changes the field of view as specified by the input."
+            },
+            {
+                "setBorderColor",
+                &skybrowser::luascriptfunctions::setBorderColor,
+                "string, int, int, int",
+                "Takes an identifier to a sky browser or a sky target and a rgb color "
+                "in the ranges [0, 255]."
+            },
+            {
+                "setScreenSpaceSize",
+                &skybrowser::luascriptfunctions::setScreenSpaceSize,
+                "string, float, float",
+                "Sets the screen space size of the sky browser to the numbers specified "
+                "by the input [x, y]."
+            },
+            {
+                "startSetup",
+                &skybrowser::luascriptfunctions::startSetup,
+                "",
+                "Starts the setup process of the sky browers. This function calls "
+                "the lua function 'sendOutIdsToBrowsers' in all nodes in the cluster."
+            },
+            {
+                "translateScreenSpaceRenderable",
+                &skybrowser::luascriptfunctions::translateScreenSpaceRenderable,
+                "string, float, float, float, float",
+                "Takes an identifier to a sky browser or sky target and the [x, y] "
+                "starting position and the [x, y] translation vector."
+             },
+        };
+
+        return res;
+    }
+
+SkyBrowserModule::SkyBrowserModule()
+    : OpenSpaceModule(SkyBrowserModule::Name)
+    , _allowMouseInteraction(AllowInteractionInfo, true)
+    , _allowCameraRotation(AllowRotationInfo, true)
+{
+    addProperty(_allowMouseInteraction);
+    addProperty(_allowCameraRotation);
+
+    // Set callback functions
+    global::callback::mouseButton->emplace_back(
+        [&](MouseButton button, MouseAction action, KeyModifier modifier) -> bool {
+            if (!_isCameraInSolarSystem || !_allowMouseInteraction) {
+                return false;
+            }
+
+            if (action == MouseAction::Press) {      
+                _isCameraRotating = false;
+                if (_mouseOnPair) {
+                    handleMouseClick(button);
+                    return true;
+                }
+                return false; 
+            }
+            else if (_interactionMode != MouseInteraction::Hover && 
```

Comment:
Since there is a return in each statement, you don't actually need the "else" here. So could be erwritten as:

```
        if (_interactionMode != MouseInteraction::Hover && action == MouseAction::Release) {

            _interactionMode = MouseInteraction::Hover;
            return true;
        }
        
        return false;
```
But this is mainly a personal preference from my side. The code get's a little cleaner :)